features/step_definitions/helpers.rb

```diff
@@ -46,14 +46,14 @@ def initialize_story_params(project, user = User.find(:first))
   story
 end
 
-def initialize_task_params(project, story, user = User.find(:first))
+def initialize_task_params(project, story, user = User.first)
```

Comment:
Assignment Branch Condition size for initialize_task_params is too high. [17.06/15]